soc/nordic/nrf54l/soc.c

```diff
@@ -47,6 +47,12 @@ LOG_MODULE_REGISTER(soc, CONFIG_SOC_LOG_LEVEL);
 #define HFXO_NODE DT_NODELABEL(hfxo)
 #endif
 
+#if DT_PROP(DT_PATH(cpus, cpu_0), clock_frequency)
+#define DEVICE_DT_CLOCK_FREQ DT_PROP(DT_PATH(cpus, cpu_0), clock_frequency)
+#elif DT_PROP(DT_PATH(cpus, cpu_1), clock_frequency)
```

Comment:
What is the logic for prioritizing `cpu_0` over `cpu_1`? (I don't know what those refer to.)

Reply:
It is just to handle the difference between cpuapp and cpuflpr. cpuapp uses cpu_0 to set clock-frequency while cpuflpr uses cpu_1. Will add a comment